apps/accounts/urls.py

```diff
@@ -3,7 +3,7 @@
 from . import views
 
 urlpatterns = [
-    url(r'^authorize/$', views.AuthorizeView.as_view(),
+    url(r'^authorize/$', views.AuthenticateView.as_view(),
```

Comment:
can the URL be changed to `authenticate/` as well? If that's too involved maybe file a bug to do it later.